docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -32,45 +32,123 @@
+.. todo::
 
     Fill out table with some guiding principles for this guide. Can refer to same table in the sex and gender guide.
 
+Introduction to race and ethnicity
+------------------------------------------
+
+Race and ethnicity are complex social constructs that have significant
+impacts on individuals' lives and health outcomes. While often used
+interchangeably, race and ethnicity are distinct concepts. Race is
+typically associated with physical characteristics such as skin color,
+facial features, and hair texture, while ethnicity encompasses cultural
+factors such as language, religion, and shared history.
+
+Historically, the concept of race has been used to justify
+discrimination, oppression, and social hierarchies. The scientific
+community has largely rejected the notion of race as a biological
+construct, recognizing that genetic variation within racial groups is
+often greater than the variation between them. However, the social,
+economic, and political implications of race persist, shaping
+individuals' experiences, opportunities, and health outcomes.
+
+Ethnicity, on the other hand, is a more fluid concept that can evolve
+over time and across generations. It is often tied to a shared cultural
+heritage, including traditions, beliefs, and values. Ethnic identities
+can be influenced by factors such as migration, acculturation, and
+intermarriage.
+
+Despite the importance of race and ethnicity in understanding health
+disparities and developing targeted interventions, there are significant
+gaps in the academic treatment of these concepts. Research often relies
+on broad racial and ethnic categories, such as "Black," "White,"
+"Hispanic," and "Asian," which can mask important heterogeneity within
+these groups. Additionally, the categories used in research may not
+align with how individuals self-identify, leading to misclassification
+and inaccurate conclusions.
+
+Data availability and collection methods also pose challenges in
+studying race and ethnicity. Many datasets lack detailed information on
+race and ethnicity, and when such data are collected, they may be based
+on observer-reported rather than self-reported information. Furthermore,
+the categories used in data collection may not be comprehensive or may
+change over time, making it difficult to compare findings across studies
+or to track trends.
+
+In the context of simulation science, it is important to recognize that
+we are often using data collected and analyzed by others. As such, we
+need to understand and adapt to the choices that they have made
+regarding race and ethnicity. This requires carefully examining the data
+sources, the categories used, and the potential limitations and biases
+inherent in the data. By doing so, we can make informed decisions about
+how to interpret and apply the data in our simulations.
+
+It is also crucial to acknowledge that the concepts of race and
+ethnicity, as understood in the United States, may not translate
```

Comment:
If our concepts of "race" and "ethnicity" are US-specific, we should mention it before this.